l10n_it_fiscalcode/i18n/am.po

```diff
@@ -0,0 +1,248 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * l10n_it_fiscalcode
+#
+# Translators:
+# OCA Transbot <transbot@odoo-community.org>, 2018
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0\n"
```

Comment:
Version 13.0, not 11.0

Reply:
It's not need to change it according to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0